include/test/test.h

```diff
@@ -215,6 +215,8 @@ static inline struct Benchmark BenchmarkStop(void)
 
 #define PARAMETRIZE if (gFunctionTestRunnerState->parameters++ == gFunctionTestRunnerState->runParameter)
 
+#define PARAMETRIZE_LABEL(f, label) if (gFunctionTestRunnerState->parameters++ == gFunctionTestRunnerState->runParameter && (MgbaPrintf_(":N%s: " f " (%d/%d)", gTestRunnerState.test->name, label, gFunctionTestRunnerState->runParameter + 1, gFunctionTestRunnerState->parameters), 1))
```

Comment:
Shouldn't this have its own documentation entry like `PARAMETRIZE` does?

Reply:
Weirdly, there's not any documentation for `include/test/test.h`-specific things, only for `include/test/battle.h`... This matters, because `PARAMETRIZE` is defined differently for each of them, and so `PARAMETRIZE_LABEL` won't work in battle tests atm.

Reply:
ah I see. Something to do properly in the future then.